apps/portal/src/app/features/file-processor/file-processor.component.ts

```diff
@@ -145,15 +145,19 @@ export class FileProcessorComponent implements OnInit, OnDestroy {
     );
   }
 
-  getSeverity(status: number): string {
-    const severityMap: { [key: number]: string } = {
+  getSeverity(
+    status: number,
+  ): 'success' | 'secondary' | 'info' | 'warning' | 'danger' | 'contrast' | undefined {
+    const severityMap: {
+      [key: number]: 'success' | 'secondary' | 'info' | 'warning' | 'danger' | 'contrast';
+    } = {
       1: 'warning',
-      2: 'primary',
+      2: 'secondary',
```

Comment:
Why was this value changed?

Reply:
We do not have a return type 'primary' for the primeng p-tag [severity]

We only have the following return types available

`severity: 'success' | 'secondary' | 'info' | 'warning' | 'danger' | 'contrast' | undefined;`

Used 'secondary' as placeholder to mantain functionality

Reply:
in the same link here, we do not have a mention for primary
Could you check this: https://primeng.org/tag#api.tag.props

Reply:
Keep primary back, it is there, they just mentioned the other values, primary is basically the default one and these valeus are always same for pretty much all components in primeNG.

You can test it by adding and checking in styles.

Reply:
Set as 'primary' by changing  function return type to any